integration/samples/same-name/specs/es5-api.ts

```diff
@@ -19,6 +21,10 @@ describe(`@sample/same-name`, () => {
   });
 
   describe(`esm5/testing/testing.js`, () => {
+    let APITesting;
+    before(() => {
+      APITesting = require('../dist/esm5/testing/testing.js');
+    });
```

Comment:
Prevents TypeScript compilation error for spec suite. Fails `before` hook in test run, if bundle files does not exist.